ci/tasks/self-updating-pipeline.yaml

```diff
@@ -0,0 +1,60 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/curl-ssl
+
+inputs:
+  - name: repository
+
+params:
+  CONCOURSE_URL:
+  CONCOURSE_TEAM:
+  CONCOURSE_PASSWORD:
+  CONCOURSE_USERNAME:
+  PIPELINE:
+  PIPELINE_NAME:
+
+run:
+  path: sh
+  args:
+  - -uec
+  - |
+    : ${CONCOURSE_TEAM:?}
+    : ${CONCOURSE_PASSWORD:?}
+    : ${PIPELINE:?}
+    : ${PIPELINE_NAME:?}
+
+    CONCOURSE_USERNAME="${CONCOURSE_USERNAME:-$CONCOURSE_TEAM}"
+    CONCOURSE_URL="${CONCOURSE_URL:-https://cd.gds-reliability.engineering}"
+
+    > fly \
+      curl -L \
```

Comment:
Should probably add `--fail` to curl args

Reply:
`curl -s google.com/404 > /dev/null; echo $?; curl -s --fail google.com/404 > /dev/null; echo $?`

Reply:
Good shout.